File: src/main/java/com/stheren/email_reader/engine/data/Sender.java
package com.stheren.email_reader.engine.data;

import java.util.HashMap;
import java.util.Map;
import com.fasterxml.jackson.annotation.JsonAnyGetter;
import com.fasterxml.jackson.annotation.JsonAnySetter;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

import javax.annotation.processing.Generated;

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder({
    "name",
    "lastName",
    "account",
    "mail"
})
@Generated("jsonschema2pojo")
public class Sender {

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("name")
    private String name;
    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("lastName")
    private String lastName;
    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("account")
    private String account;
    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("mail")
    private String mail;
    @JsonIgnore
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    /**
     * No args constructor for use in serialization
     * 
     */
    public Sender() {
    }

    /**
     * 
     * @param lastName
     * @param mail
     * @param name
     * @param account
     */
    public Sender(String name, String lastName, String account, String mail) {
        super();
        this.name = name;
        this.lastName = lastName;
        this.account = account;
        this.mail = mail;
    }

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("name")
    public String getName() {
        return name;
    }

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("name")
    public void setName(String name) {
        this.name = name;
    }

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("lastName")
    public String getLastName() {
        return lastName;
    }

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("lastName")
    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("account")
    public String getAccount() {
        return account;
    }

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("account")
    public void setAccount(String account) {
        this.account = account;
    }

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("mail")
    public String getMail() {
        return mail;
    }

    /**
     * 
     * (Required)
     * 
     */
    @JsonProperty("mail")
    public void setMail(String mail) {
        this.mail = mail;
    }

    @JsonAnyGetter
    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    @JsonAnySetter
    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: src/main/kotlin/com/stheren/email_reader/data/Sender.kt
package com.stheren.email_reader.data

import com.fasterxml.jackson.annotation.*
import javax.annotation.processing.Generated

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder("name", "lastName", "account", "mail")
class Sender {
    @get:JsonProperty("name")
    @set:JsonProperty("name")
    @JsonProperty("name")
    lateinit var  name: String

    @get:JsonProperty("lastName")
    @set:JsonProperty("lastName")
    @JsonProperty("lastName")
    lateinit var  lastName: String

    @get:JsonProperty("account")
    @set:JsonProperty("account")
    @JsonProperty("account")
    lateinit var  account: String

    @get:JsonProperty("mail")
    @set:JsonProperty("mail")
    @JsonProperty("mail")
    lateinit var  mail: String

    @JsonIgnore
    private val additionalProperties: MutableMap<String, Any> = HashMap()

    constructor() {}

    constructor(name: String, lastName: String, account: String, mail: String) : super() {
        this.name = name
        this.lastName = lastName
        this.account = account
        this.mail = mail
    }

    @JsonAnyGetter
    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    @JsonAnySetter
    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}